src/handlers/settlement-window-close.js

```diff
@@ -7,16 +7,23 @@ const handler = (router, routesContext) => {
         const { startDate: fromDateTime, endDate: toDateTime } = ctx.request.body;
 
         try {
-            await axios.post(`${routesContext.config.settlementManagementEndpoint}/close-window`, {});
-            ctx.response.status = 200;
+            const resp = await axios.post(`${routesContext.config.externalSettlementsEndpoint}/settlement-windows/current/close`, {});
+            if (resp.status === 202) {
+                ctx.response.status = 200;
+            } else {
+                ctx.response.status = 502;
+            }
         } catch (err) {
-            ctx.response.status = 500;
+            ctx.response.status = 502;
         } finally {
-            ctx.response.body = await handlerHelpers.getSettlementWindows(
-                routesContext,
-                fromDateTime,
-                toDateTime,
-            );
+            // this setTimeout is introduced to give TMF enough time to close the window
+            setTimeout(async () => {
```

Comment:
Is using `await` here possible? I feel like this causes a race condition if not.